profiles/templates/profiles/view_profile.html

```diff
@@ -15,4 +15,72 @@ <h3>Created Events:</h3>
 {% else %}
   <p>No events created yet.</p>
 {% endif %}
+
+<div class="col-md-4 my-3">
+  {% if user.is_authenticated %}
+      {% if user != user_profile.user %}
+          <!-- The user is not the creator of the event -->
+          {% if join_status == PENDING %}
```

Comment:
The functionality works great! Can we replace all the original "join_event" related variable name to the real "add-friend" variable names? I know it doesn't affect the code execution, but right now it's really confusing when we read the code. Thank you!

Reply:
Made the changes. :)